stumpy/stumped.py

```diff
@@ -248,13 +248,20 @@ def stumped(dask_client, T_A, m, T_B=None, ignore_trivial=True, normalize=True,
                 T_B_subseq_isconstant_future,
                 diags_futures[i],
                 ignore_trivial,
+                1,
             )
         )
 
     results = dask_client.gather(futures)
-    profile, indices = results[0]
+    profile, indices, profile_L, indices_L, profile_R, indices_R = results[0]
+
+    profile = np.c_[profile, profile_L, profile_R]
+    indices = np.c_[indices, indices_L, indices_R]
+
     for i in range(1, len(hosts)):
-        P, I = results[i]
+        P, I, PL, IL, PR, IR = results[i]
+        P = np.c_[P, PL, PR]
```

Comment:
Again, we only need `P`. `PL` and `PR` are not needed

Reply:
Could you please take a look at `for-loop` block in line 265? Because, if  I understand correctly, you are updating profile and indices of left and right profiles based on the condition that depends on the left and right matrix profile. So, shouldn't we consider `PL` and `PR`?

Reply:
Ahhh, you're right! I totally forgot about that. Indeed, we were recording the `PL` and `PR` for each worker so that we can choose the right `IL` and `IR`

Reply:
Please use `column_stack` instead then

Reply:
Sure. I will revise it.